modules/alb/versions.tf

```
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 0.14.0"
```

```suggestion
  required_version = ">= 1.0.0"
```
Could update this or leave it - usually for components we keep it up to date and modules we leave the minimum